config/jest.config.js

```diff
@@ -10,6 +10,16 @@ const defaults = {
     escapeString: true,
     printBasicPrototype: true
   },
+  transform: {
+    '^.+\\.tsx?$': [
+      'ts-jest',
+      {
+        diagnostics: {
+          warnOnly: process.env.TEST_ENV === 'ci' ? false : true
```

Comment:
```suggestion
          warnOnly: process.env.TEST_ENV !== 'ci'
```

[nit] No need for the ternary since the `===`/`!==` returns a boolean.

Reply:
Good call! Updated